pkg/bindings/generator/generator.go

```diff
@@ -239,7 +236,7 @@ func main() {
 				closed = true
 
 				// go fmt file
-				gofmt := exec.Command("gofmt", "-w", "-s", out.Name())
+				gofmt := exec.Command("go", "fmt", out.Name())
```

Comment:
why did you remove this?

Reply:
gofmt is not available in path on a lot of ci/cd images, go fmt seems to work, so I switched to it.

```
 go fmt [-n] [-x] [packages]

Fmt runs the command 'gofmt -l -w' on the packages named
by the import paths. It prints the names of the files that are modified.

For more about gofmt, see 'go doc cmd/gofmt'.
For more about specifying packages, see 'go help packages'.

The -n flag prints commands that would be executed.
The -x flag prints commands as they are executed.

The -mod flag's value sets which module download mode
to use: readonly or vendor. See 'go help modules' for more.

To run gofmt with specific options, run gofmt itself.

See also: go fix, go vet.
```